libraries/AP_HAL_ChibiOS/hwdef/CUAV-Nora/hwdef-bl.dat

```diff
@@ -0,0 +1,58 @@
+# hw definition file for processing by chibios_hwdef.py
+# for H743 bootloader
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# USB setup
+USB_VENDOR 0x0483 # ST
```

Comment:
you should remove the USB_VENDOR and USB_PRODUCT lines, we now set these automatically

Reply:
Ok